teamwork/apps/profiles/models.py

```diff
@@ -204,6 +236,11 @@ class Profile(models.Model):
     known_skills = models.ManyToManyField(Skills, related_name="known", default="")
     learn_skills = models.ManyToManyField(Skills, related_name="learn", default="")
 
+#------------------------------------------------------------------------------------
+    known_techs = models.ManyToManyField(Techs, related_name="known", default="")
```

Comment:
Need to change the related name since `known` and `learn` are already used by skills. Please change related_name to `known_technologies` and `learn_technologies`